drivers/spi/spi_ll_stm32.c

```diff
@@ -729,8 +729,7 @@ static int transceive_dma(const struct device *dev,
 		/* wait until hardware is really ready */
 		while (LL_SPI_IsActiveFlag_BSY(spi) == 1) {
 		}
-
-		LL_SPI_Disable(spi);
```

Comment:
Btw, aren't lines 689 to 691 superfluous? (At this stage, cs nor dma are enabled: the transaction has yet to start)

Reply:
Probably yes but this PR solves only HW CS problem. I don't know what was the idea of the original author.

Reply:
I think that code ended up there after quit a bit of conversation in the PR that changed that code, I guess nobody had a use case/hardware that used HW CS :-/ I will test (but I won't have time until in the weekend) it on the boards I have, and if those all work I guess it should be OK.

Reply:
@tbursztyka a quick test shows that line 689 and 690 can be removed, but line 691 `LL_SPI_Disable(spi);` should be changed to `LL_SPI_Enable(spi);` and than line 713 can also be removed.

Reply:
nice, can you make a pr?

Reply:
@LuxLuthor could you add those changes mentioned by me to this PR, or do you want me to make a new combined PR with those changes and your fix ?

Reply:
@erwango the question was if cleanup the extra enable call in this PR, or if we make two "mini" PR's

Reply:
@lowlander @LuxLuthor, as long as this is done in 2 commits that are properly documented and current PR description is updated, this can be done here.

Reply:
@erwango the fix is to remove the disable from the loop, so the enable in the loop makes no sense anymore, and can only be done one time before the loop. so those two changes belong in one (this) commit ? The removing of the dmadisable in front of the loop could be the second commit ?

Reply:
@erwango @lowlander, Yep, I will add these changes in this PR.

Reply:
@erwango On the other hand maybe just merge this commit and make a cleanup PR, so @Luxluthor doesn't have to change this PR/commit ? I can make a new PR with the cleanup.

Reply:
@erwango @tbursztyka should @luxluthor make one commit with removing the SPI disable/enable out of the loop and one removing the two dmadisables in front of the loop ?